### PR TITLE
Enhancement: Rename examples folder

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ plus its curl and json extensions. See below the `require` section of project's
 
 ## Pushing Messages
 
-*Note: For more code examples, see [Example](Example) folder in the root of the project. You may also generate and see code documentation.*
+*Note: For more code examples, see [Example](examples) folder in the root of the project. You may also generate and see code documentation.*
```
Comment (slunak) [inline README.md:73]:
```suggestion
*Note: For more code examples, see [examples](examples) folder in the root of the project. You may also generate and see code documentation.*
```

**Comment by slunak** [comment]
Probably needs renaming folder in CI config too :)

**Comment by OskarStark** [comment]
nvm, got it

**Comment by slunak** [comment]
Thank you :)

**Review by slunak — APPROVED** [review]

**Pull request merged**